x-pack/solutions/security/plugins/security_solution/public/data_view_manager/components/data_view_picker/index.tsx

```diff
@@ -21,115 +21,117 @@ import { DATA_VIEW_PICKER_TEST_ID } from './constants';
 import { useManagedDataViews } from '../../hooks/use_managed_data_views';
 import { useSavedDataViews } from '../../hooks/use_saved_data_views';
 
-export const DataViewPicker = memo((props: { scope: DataViewManagerScopeName }) => {
-  const dispatch = useDispatch();
-  const selectDataView = useSelectDataView();
-
-  const {
-    services: { dataViewEditor, data, dataViewFieldEditor, fieldFormats },
-  } = useKibana();
-  const closeDataViewEditor = useRef<() => void | undefined>();
-  const closeFieldEditor = useRef<() => void | undefined>();
+export const DataViewPicker = memo(
+  (props: { scope: DataViewManagerScopeName; disabled?: boolean }) => {
```

Comment:
nit: Please extract props definition to a separate interface.

Reply:
Prop definition added in [this PR](https://github.com/elastic/kibana/pull/218183/files#diff-13353d8539cd2acd58a5e3ed139262a00dbe62b6b51ffbe31d417b991c22f2d4R24), depending which one goes first :)